tests/core/editable/manual/insert.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: bug, 4.12.0, 2813
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, sourcearea
+
+## For each insert button
+
+1. After each case use `reset editor` button.
+
+1. Select word `bar`.
+
+1. Press button.
+
+## Expected
+- `Insert text` button: Inserted word `text` replaces selected word inside border.
+- Any other button : there is one red border on the left of inserted `div`.
```

Comment:
Not true for `insertElement/insertElementInotRange` functions where `div` is placed below rectangle. I would suggest rewording manual test a bit to make it more general:

```
## For each insert button

1. After each case use `reset editor` button.

1. Select the word `bar`.

1. Press button.

## Expected
- `Insert text` button: `text` replaces selected word inside the rectangle
- Any other button: `div` replaces selected word outside the rectangle

## Unexpected
Clicked button causes the creation of an additional rectangle

**NOTE:** Insert HTML functions inserts `div` element as a plain text. Caused by [#3042](https://github.com/ckeditor/ckeditor-dev/issues/3042) issue.
```

In that case, #3042 issue won't invalidate the manual test.